gramps/plugins/export/exportcsv.py

```diff
@@ -317,9 +251,9 @@ def write_csv(self, *items):
     def export_data(self):
         self.dirname = os.path.dirname (self.filename)
         try:
-            self.g = open(self.filename,"w")
-            self.fp = open(self.filename, "wb")
-            self.g = UnicodeWriter(self.fp)
+            self.fp = open(self.filename, "w", encoding='utf_8_sig',
```

Comment:
This change is on the output side though. The BOM only came about on Windows because they picked UTF-16 which could have two orders, and then it percolated to UTF-8, too. But on Linux, no-one bothers with a useless BOM because the whole OS went UTF-8 wholesale.